src/Controller/EventController.php

```diff
@@ -75,6 +76,7 @@ public function searchEvent(Request $request, EventRepository $eventRepository)
                 'events' => [],
                 'form' => $form->createView(),
                 'years' => $years,
+                'showResult' => false,
```

Comment:
@daria-sieroshtan 
you don't need this var
In template do `if (events | length > 0)`

Reply:
I guess it won't work this way. I need different behavior for "form is not submitted" and "form submitted and result is empty array", and if I only check on length, this two cases are similar.